src/manifolds/Unitary.jl

```diff
@@ -0,0 +1,31 @@
+
+@doc raw"""
+    const UnitaryMatrices{n} = AbstarctUnitaryMatrices{n,ℂ,AbsoluteDeterminantOneMatrices}
+
+The manifold ``U(n)`` of ``n×n`` complex matrices
+
+``p^{\mathrm{H}p = \mathrm{I}_n,``
+
+where ``\mathrm{I}_n`` is the ``n×n`` identity matrix.
+An alternative characterisation is that ``\lVert \det(p) \rVert = 1``.
```

Comment:
```suggestion
Such matrices `p` have a property that ``\lVert \det(p) \rVert = 1``.
```
Except for n=1 there are matrices of absolute determinant 1 that are not unitary.